src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,19 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <article className={cn('TodoInfo', {
+      'TodoInfo--completed': completed,
+    })}
+    >
```

Comment:
```suggestion
    <article className={cn(
      'TodoInfo', 
      {
      'TodoInfo--completed': completed,
      }
    )}
    >
```
i think it will be more readable